xwiki-platform-core/xwiki-platform-oldcore/src/main/resources/ApplicationResources.properties

```diff
@@ -3713,7 +3713,7 @@ extensions.uninstall.cleanPages.label=Delete unused wiki pages?
 extensions.uninstall.cleanPages.hint=The following wiki pages are not needed any more so it should be safe to delete them. Unselect the ones that you wish to keep. The wiki pages that have modifications are left unselected so that you don't loose your changes. Select them if those changes are not important.
 extensions.uninstall.cleanPages.selectedCount={0} / {1} pages selected
 extensions.search.submit=Search
-extensions.search.tip=search extension...
+extensions.search.tip=Search extension…
```

Comment:
Are you sure that the capitalization of this string should be changed? I'm not saying that it is wrong, I'm just asking if you've checked where it is used and that this uppercase letter is really correct in that context.